include/lz4.h

```diff
@@ -219,6 +219,16 @@ int           LZ4_freeStream (LZ4_stream_t* streamPtr);
  */
 int LZ4_loadDict (LZ4_stream_t* streamPtr, const char* dictionary, int dictSize);
 
+/* LZ4_stream_preserve
+ * Use this function to preserve a stream after loading a dictionary, so it can be rapidly reloaded.
+ */
+int LZ4_stream_preserve(LZ4_stream_t* stream_, int** buf);
```

Comment:
Ah, the buf is malloced within `LZ4_stream_reserve`. It's worth saying that the user must `free()` this when they are done.